contracts/DKVDaggerHashimoto.sol

```diff
@@ -277,32 +358,57 @@ contract DecentralizedKVDaggerHashimoto is DecentralizedKV {
         uint256 startShardId,
         uint256 shardLenBits,
         bytes32 hash0,
+        address miner,
         bytes32[][] memory proofsDim2,
         bytes[] memory maskedData
     ) internal view returns (bytes32) {
-        require(maskedData.length == randomChecks, "data vs checks: length mismatch");
-        require(proofsDim2.length == randomChecks, "proofs vs checks: length mismatch");
-        uint256 maxKvSize = 1 << maxKvSizeBits;
+        require(
+            maskedData.length == randomChecks,
+            "data vs checks: length mismatch"
+        );
+        require(
+            proofsDim2.length == randomChecks,
+            "proofs vs checks: length mismatch"
+        );
+        // calculate the number of chunks range of the sample check
         uint256 rows = 1 << (shardEntryBits + shardLenBits + chunkLenBits);
```

Comment:
Minor: `row` can be constant. `uint256 constant row`

Reply:
hh, I think this is invalid solidity grammar.